lib/scholar/metrics/ranking.ex

```diff
@@ -94,4 +94,42 @@ defmodule Scholar.Metrics.Ranking do
         end
     end
   end
+
+  @doc """
+  Computes the NDCG based on true relevance scores (`y_true`) and their respective predicted scores (`y_score`).
```

Comment:
```suggestion
Computes the normalized discounted cumulative gain (NDCG) based on true relevance scores `y_true` and their respective predicted scores `y_score`.
```

Let's include the full metric name as well - you never know how people might search for it on hexdocs.